index.js

```diff
@@ -1,7 +1,10 @@
-var window              = require('global');
-var MockXMLHttpRequest  = require('./lib/MockXMLHttpRequest');
-var real                = window.XMLHttpRequest;
-var mock                = MockXMLHttpRequest;
+var Promise                 = require('lie');
+var window                  = require('global');
+var MockXMLHttpRequest      = require('./lib/MockXMLHttpRequest');
+var real                    = window.XMLHttpRequest;
+var mock                    = MockXMLHttpRequest;
+
+window.xhrMockProxyHandler  = require('./lib/xhr-mock-proxy-handler');
```

Comment:
I wasn't sure what you mentioned in issue. So I agree it's better, let's create a separate NPM package for this. Could we refer to it in README after it will be created?